### PR TITLE
defend against something else adding callback names to the FT namespace

### PR DESCRIPTION
**Pull request by wheresrhys**
@AlbertoElias

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -2,8 +2,18 @@ const jsonpCallbackNames = [];
...
 	const base = 'jsonpCallback';
-	const callbackName = `${base}_${jsonpCallbackNames.length + 1}`;
-	jsonpCallbackNames.push(callbackName);
+	let callbackName;
+	let isUniqueName = false;
+	while (!isUniqueName) {
+		callbackName = `${base}_${jsonpCallbackNames.length + 1}`;
+		isUniqueName = !window.FT[callbackName];
+		if (!isUniqueName) {
+			jsonpCallbackNames.push(null);
```
Comment (AlbertoElias) [inline main.js:11]: why even bother pushing null?

Comment (wheresrhys) [inline main.js:11]: to increment length

Comment (wheresrhys) [inline main.js:11]: In hindsight, using timestamp + random string as callback name woudl be simpler and resilient enough

**Comment by AlbertoElias** [comment]
Looks good, but tests are failing

**Pull request closed without merging**